src/index.js

```diff
@@ -75,6 +75,7 @@ function createAcceptButton(attributes = {}, events = {}) {
         backgroundColor: attributes.btnbgcolor,
         borderColor: attributes.btnbgcolor,
         color: attributes.btntextcolor,
+        borderRadius: attributes.btntBorderRadius,
```

Comment:
@iannmuller0 , os atributos do HTML são sempre em caixa baixa:

```suggestion
        borderRadius: attributes.btnborderradius,
```